html/settings/_settings.scss

```diff
@@ -5,920 +5,1768 @@
+/// The background color of the Modal.
 $sprk-modal-color: $sprk-white !default;
+/// The maximum width of the Modal.
 $sprk-modal-max-width: 43.75rem !default;
+/// The maximum height of the Modal.
 $sprk-modal-height: 70% !default;
+/// The point at which the Modal
+/// moves farther down the viewport from the top.
 $sprk-modal-breakpoint-small: 25rem !default;
+/// The point at which the Modal moves even
+/// farther down the viewport from the top than
+// it does at the small breakpoint.
 $sprk-modal-breakpoint-medium: 37.5rem !default;
+/// The box shadow on the Modal.
 $sprk-modal-shadow: 0 4px 5px rgba(0, 0, 0, 0.6) !default;
+/// The border radius on the Modal.
 $sprk-modal-border-radius: 8px !default;
 
 //
 // Masthead
 ///
+
+/// The width of the menu icon on narrow viewports in the Masthead.
 $sprk-masthead-menu-icon-width: $sprk-icon-l !default;
+/// The height of the menu icon on narrow viewports in the Masthead.
 $sprk-masthead-menu-icon-height: $sprk-icon-l !default;
+/// The background color of the Masthead.
 $sprk-masthead-bg-color: $sprk-white !default;
+/// The padding of the Masthead content, menu, and branding sections.
 $sprk-masthead-content-padding: $sprk-space-s !default;
+/// The padding top around the content items in the
+/// content section of the Masthead.
 $sprk-masthead-content-item-padding-top: $sprk-space-s !default;
+/// The padding bottom around the content
+/// items in the content section of the Masthead.
 $sprk-masthead-content-item-padding-bottom: $sprk-space-s !default;
+/// The padding right around the content items in the
+/// content section of the Masthead.
 $sprk-masthead-content-item-padding-right: $sprk-space-s !default;
+/// The padding left around the content items in the
+// content section of the Masthead.
 $sprk-masthead-content-item-padding-left: $sprk-space-s !default;
+/// The padding around the content
+/// section in the Masthead on wide viewports.
 $sprk-masthead-content-padding-wide: $sprk-space-m !default;
+/// The padding top of the content
+/// items in the content section in the Masthead on wide viewports.
 $sprk-masthead-content-item-padding-top-wide: $sprk-space-s !default;
+/// The padding bottom of the content
+/// items in the content section in the Masthead on wide viewports.
 $sprk-masthead-content-item-padding-bottom-wide: $sprk-space-s !default;
+/// The padding right of the content
+/// items in the content section in the Masthead on wide viewports.
 $sprk-masthead-content-item-padding-right-wide: $sprk-space-m !default;
+/// The padding left of the content
+/// items in the content section in the Masthead on wide viewports.
 $sprk-masthead-content-item-padding-left-wide: $sprk-space-m !default;
+/// The visited state color of Masthead links.
 $sprk-masthead-link-visited-color: $sprk-black !default;
+/// The hover state color of Masthead links.
 $sprk-masthead-link-hover-color: $sprk-green !default;
+/// The font weight of Masthead links.
```

Comment:
Do we want to capitalize the `S` in Stepper Step? Since it is an actual component.

Reply:
I only capitalized the components in the DS. Technically on the backend, we have made the step a component too but its not something we have documented as an available component in the DS on its own.